content/sdk/10.js/00.ethers/20.v6/00.getting-started.md

```diff
@@ -1,69 +1,154 @@
 ---
 title: "Getting Started"
 description: "Explore zkSync-ethers SDK version 6"
-tags: ["zksync", "sdks", "zkSync Era SDK", "ethereum", "era network", "javascript", "v6"]
+tags: ["zksync", "sdks", "zkSync Era SDK", "ethereum", "era network", "javascript", v6"]
 ---
 
-While most of the existing SDKs should work out of the box, deploying smart contracts or using unique zkSync features,
-like account abstraction, requires providing additional fields to those that Ethereum transactions have by default.
+## Overview
 
-To begin, it is useful to have a basic understanding of the types of objects available and what they are
-responsible for, at a high level:
+This guide will help you interact with the zkSync Era blockchain using the zkSync JavaScript SDK. We'll walk through
```

Comment:
Let's clarify this is the zksync-ethers SDK, not the javascript sdk. There may be more javascript libraries we will support later on.